.pylintrc

```diff
@@ -60,7 +66,70 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-#disable=
+disable=
+    # These are disabled by default:
+    import-star-module-level,
+    old-octal-literal,
+    oct-method,
+    print-statement,
+    unpacking-in-except,
+    parameter-unpacking,
+    backtick,
+    old-raise-syntax,
+    old-ne-operator,
+    long-suffix,
+    dict-view-method,
+    dict-iter-method,
+    metaclass-assignment,
+    next-method-called,
+    raising-string,
+    indexing-exception,
+    raw_input-builtin,
+    long-builtin,
+    file-builtin,
+    execfile-builtin,
+    coerce-builtin,
+    cmp-builtin,
+    buffer-builtin,
+    basestring-builtin,
+    apply-builtin,
+    filter-builtin-not-iterating,
+    using-cmp-argument,
+    useless-suppression,
+    range-builtin-not-iterating,
+    suppressed-message,
+    no-absolute-import,
+    old-division,
+    cmp-method,
+    reload-builtin,
+    zip-builtin-not-iterating,
+    intern-builtin,
+    unichr-builtin,
+    reduce-builtin,
+    standarderror-builtin,
+    unicode-builtin,
+    xrange-builtin,
+    coerce-method,
+    delslice-method,
+    getslice-method,
+    setslice-method,
+    input-builtin,
+    round-builtin,
+    hex-method,
+    nonzero-method,
+    map-builtin-not-iterating,
+    # These are disabled in openshift-ansible:
+    # disabling rules is not to be reported as a violation
+    locally-disabled,
+    # some files are to be ignored
+    file-ignored,
+    # it is expected to have TODO notes
+    fixme,
+    # sometimes it's better to have no docstrings than placeholders or incorrect text
+    missing-docstring,
```

Comment:
@rhcarvalho I think we should probably discuss these pylint disables at the architecture meeting this week. Personally, I would like to see us leveraging pytest's doctest support: http://doc.pytest.org/en/latest/doctest.html, but we can do that with or without docstring enforcement, it would just make it a bit more 'standard' if we were enforcing docstrings.

One thing that I'm wondering about is if we should have a different set of standards for tests than we do for other python files. I don't see any reason to enforce docstrings, invalid-name, wrong-import-position, etc for tests.